site/docs/design_tokens/colour.mdx

```diff
@@ -87,10 +89,58 @@ Status colours are reserved exclusively for indicating valid, invalid or informa
 | <ColorBox color="var(--color-black)"/> | <ColorBox color="var(--color-black-90)"/> | <ColorBox color="var(--color-black-80)"/> | <ColorBox color="var(--color-black-70)"/> | <ColorBox color="var(--color-black-60)"/> | <ColorBox color="var(--color-black-50)"/> | <ColorBox color="var(--color-black-40)"/> | <ColorBox color="var(--color-black-30)"/> | <ColorBox color="var(--color-black-20)"/> | <ColorBox color="var(--color-black-10)"/> | <ColorBox color="var(--color-black-5)"/> | <ColorBox color="var(--color-white)" /> |
 
 
+## Accessible colour combinations
+<p>Here you can find some recommended colour combinations for text and other informative elements. The examples are sorted in order of their contrast ratio. For more examples of accessibile colour combinations, see the <Link href="https://share.goabstract.com/28352b8b-9c51-4346-b249-ff062c5da561" external>Colour accessibility examples - Abstract collection</Link>.</p>
+
+- **AA** and **AAA** means you can safely use the color combination on all text sizes
+- **AA for large text** means the text size should be at least 18 pt for regular weight or 14 pt for bold weight
+- **Black 90** is the default text colour in HDS
+
+### Brand colour combinations
+
+| Brand Colour      | Text Colour   | Colour contrast   | Example           |
+| ----------------- | ------------- | ----------------- | ----------------- |
+| **Engel**         | black 90      | AAA (14,3 : 1)    | <ColorContrast color="var(--color-black-90)" background="var(--color-engel)">AAA</ColorContrast> |
```

Comment:
> AAA (14,3 : 1) 

Näihin voisi kokeilla kontrastinumeroihin kooditageja (niillä uusilla tyylittelyillä), jos se nostaisi numeroja paremmin esiin.

Reply:
Muutin kooditageiksi, mutta en saanut tuotua kooditagien tyyliä tähän branchiin joten en tiedä miltä näyttää. Muutin samalla AAA ja AA tason merkinnät boldiksi ja AA for large textit vaaleammiksi.

Reply:
Noi A-tasot erottuu nyt ihan kivasti toisistaan, hyvä. 

Luulen, että uusilla kooditageilla noi näyttää hyvältä kyllä.